README.md

```diff
@@ -56,8 +56,63 @@ message Baz {
 }
 ```
 
-`protoc --bq-schema_out=. foo.proto` will generate a file named `foo/bar_table.schema`.
-The message `foo.Baz` is ignored because it doesn't have option `gen_bq_schema.bigquery_opts`.
+### Example - Policy Tags
+Support exists for specifying a [BigQuery Policy Tag](https://cloud.google.com/bigquery/docs/column-level-security-intro) on a field.
+
+Suppose that we have the following `foo.proto` that has `policy_tags` set on the `first_name` field:
+
+```protobuf
+syntax = "proto3";
+package foo;
+import "bq_table.proto";
+import "bq_field.proto";
+
+message Bar {
+  option (gen_bq_schema.bigquery_opts).table_name = "bar_table";
+
+  required string user = 1;
+  optional string first_name = 2 [(gen_bq_schema.bigquery).policy_tags = "pii"];
+}
+```
+
+`protoc --bq-schema_out=. foo.proto` will generate a file named `foo/bar_table.schema`. The table `bar_table` is defined in `bar_table.schema` and the field with name `first_name` includes the `policyTags` json marking it as a field which is subject to some form of column-level security.
+
+```json
+{
+  "name": "first_name",
+  "type": "STRING",
+  "mode": "NULLABLE",
+  "policyTags": {
+   "names": [
+    "pii"
+   ]
+  }
+ }
+```
+
+Policy tags must take the form `projects/project-id/locations/location/taxonomies/taxonomy-id/policyTags/policytag-id` and so users can either specify this form in the `.proto` file, or use an alias which refers to an entry in a config file, e.g.
+
+`policy_tags.env`
+```dotenv
+pii=projects/my-project/locations/EU/taxonomies/142965068551231978/policyTags/projects/uw-partner-dev/locations/eu/taxonomies/142935068551331978/policyTags/9114836628731764131
+```
+
+`protoc --bq-schema_out=policy_tags_config_path=policy_tags.env:. foo.proto` will generate a file named `foo/bar_table.schema`, but in this instance the BigQuery field is generated like so:
+
+```json
+{
+  "name": "first_name",
+  "type": "STRING",
+  "mode": "NULLABLE",
+  "policyTags": {
+   "names": [
+    "projects/my-project/locations/EU/taxonomies/142965068551231978/policyTags/projects/uw-partner-dev/locations/eu/taxonomies/142935068551331978/policyTags/9114836628731764131"
```

Comment:
Are these going to be shared between dev and prod? If not then we would have to generate the schemas twice, what is not ideal. In that case it might be worth putting aliases in the generated files and let the schema applier to resolve them to correct versions for prod and dev...

Reply:
Very good point. The policyTags are going to be environment specific. I guess we don't really want to generate those schemas twice.
I'll remove the config code and let schema applier handle the environmental differences as you suggest.